docker/Dockerfile

```diff
@@ -17,10 +19,10 @@ COPY ./pages ./
 WORKDIR /usr/src/app/utils
 COPY ./utils ./
 WORKDIR /usr/src/app
-COPY app.py config.py app_sidebar_collapsible.py assets globals.py globalsUpdater.py Procfile ./
+COPY app.py app_sidebar_collapsible.py globals.py globalsUpdater.py Procfile ./
 
 WORKDIR /usr/src/app/assets
-COPY assets/style.css ./
+COPY assets/ ./
```

Comment:
Yes, I decided to change it so the assets/ directory is copied over correctly, which wasn't the case earlier.

I came across this issue when trying to read in the `openpath logo image` because it was being copied in root app directory and not in `assets` due to the `assets` directory being there in the `COPY` layer. 

The reason is that if any folder is mentioned in the COPY layer in a Dockerfile, it copies contents of the directory (and not the directory) in the root app directory.
Check the comment by user Claudiu [here](https://stackoverflow.com/a/30316600).

-----

Thus in the existing code too, the `COPY app.py assets...` layer was copying contents of assets, i.e. `style.css` to the working directory set before the command - `/usr/src/app`.
Hence, to retain assets/ directory structure which contains style.css, another COPY layer was used to copy style.css separately.

To fix this duplication, I removed assets from the first `COPY` layer and mentioned only `assets/` in the 2nd `COPY` layer. This resulted in maintaining the directory structure of assets and did not result in copying contents of `assets/ to /usr/src/app.`